samples/nvidia-resnet/install_kubeflow_and_dependencies.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Because of some GPU dependencies for Docker runtime and Kubernetes, we wanted to provide a one-time installation script when someone tries this example in a new system. After that, for every new pipeline, the user only has to run the `builld_pipeline.py` again, which just rebuilds the images for each pipeline component and recompiles the pipeline definition. If there're better practices for this process, please let me know.

Reply:
/cc @IronPan 

Yang, is there a way we could modify the launcher scripts so that the NVIDIA use case is supported?

Reply:
Do you have any update on this? Also which launcher scripts are you referring to?